src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -112,15 +137,16 @@ export const Configurator = ({
                     text: field.displayName || field.name,
                     value: field.name,
                   })),
-                  { text: '-----', value: '', disabled: true },
-                  { text: 'No time field', value: undefined },
+                  { text: '-----', value: '-----', disabled: true },
+                  { text: noTimeFilter, value: noTimeFilter },
```

Comment:
`noTimeFilter` is storing user facing text, could `text` and `value` be separated? text should have i18n and value should be a fixed key

Reply:
Keeping the value as an empty string or undefined would result in some weird behaviors when the `value` prop for `OuiSelectable` is undefined, which is why I made them the same.